protobuf/flow/entities/event.proto

```diff
@@ -12,3 +12,11 @@ message Event {
   uint32 event_index = 4;
   bytes payload = 5;
 }
+
+
+message EventProof {
+  uint32 version = 1;
+  bytes event_hash = 2; // a 32 bytes storing the hash of event 
+  uint32 hash_value_counts = 3; // number of interim hash values provided
+  repeated bytes hash_values = 4; // a series of 32-byte hash values (starting from leaf to the top of the trie; first value is a leaf neighbour, and last value is a children of the root node)
+}
```

Comment:
I'm not very familiar with `EventProof`, etc yet. so my comment is very limited.

What do you think about reordering the fields to reduce the generated struct size by 8 bytes?

```Go
message EventProof {
  uint32 version = 1;
  uint32 hash_value_counts = 2; // number of interim hash values provided
  bytes event_hash = 3; // a 32 bytes storing the hash of event 
  repeated bytes hash_values = 4; // a series of 32-byte hash values (starting from leaf to the top of the trie; first value is a leaf neighbour, and last value is a children of the root node)
}
```